memo_app.rb

```diff
@@ -16,26 +16,29 @@ def connect_db
 configure do
   result = connect_db.exec("SELECT * FROM information_schema.tables WHERE table_name = 'memos'")
   if result.values.empty?
-    connect_db.exec('CREATE TABLE memos (
+    connect_db.exec(<<CREATE_TABLE_SQL)
+      CREATE TABLE memos (
         id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
         title VARCHAR(255) NOT NULL,
         content TEXT,
         created_at TIMESTAMP WITH TIME ZONE NOT NULL,
         updated_at TIMESTAMP WITH TIME ZONE NOT NULL
-      )')
+      );
+CREATE_TABLE_SQL
   end
 end
 
 def load_memos
   connect_db.exec('SELECT * FROM memos ORDER BY updated_at DESC')
 end
 
-def add_memo(title, content)
-  connect_db.exec_params('INSERT INTO memos(title, content, created_at) VALUES ($1, $2, $3);', [title, content, DateTime.now])
+def add_memo(params)
+  connect_db.exec_params('INSERT INTO memos(title, content, created_at) VALUES ($1, $2, $3);', [params['title'], params['content'], Time.now])
 end
 
-def update_memo(title, content, id)
-  connect_db.exec_params('UPDATE memos SET title = $1, content = $2, updated_at = $3 WHERE id = $4;', [title, content, DateTime.now, id])
+def update_memo(params)
+  connect_db.exec_params('UPDATE memos SET title = $1, content = $2, updated_at = $3 WHERE id = $4;',
+                         [params['title'], params['content'], Time.now, params['id']])
```

Comment:
```suggestion
  connect_db.exec_params(
    'UPDATE memos SET title = $1, content = $2, updated_at = $3 WHERE id = $4;',
    [params['title'], params['content'], Time.now, params['id']]
  )
```

好みの要素も大きいですが、こんな感じにした方が見やすいかも

Reply:
ぱっと見の見やすさは大事ですね。